github/repos_collaborators.go

```diff
@@ -49,6 +49,35 @@ func (s *RepositoriesService) IsCollaborator(owner, repo, user string) (bool, *R
 	return isCollab, resp, err
 }
 
+// RepositoryPermissionLevel represents the permission level an organization
+// member has for a given repository.
+type RepositoryPermissionLevel struct {
+	// Possible values: "admin", "write", "read", "none"
+	Permission string `json:"permission"`
+
+	User *User `json:"user"`
```

Comment:
Was there a reason to avoid `,omitempty` option for both `User` and `Permission`, unlike all other structs we have?